services/compute/compute.go

```diff
@@ -230,6 +230,31 @@ type NetworkProfile struct {
 	NetworkInterfaces *[]NetworkInterfaceReference `json:"networkinterfaces,omitempty"`
 }
 
+type GuestAgentProfile struct {
+	// Enabled - Specifies whether guest agent should be enabled on the virtual machine.
+	Enabled *bool `json:"enabled,omitempty"`
+}
+
+type InstanceViewStatus struct {
+	// Code - READ-ONLY; The status code, which only appears in the response.
+	Code string `json:"code,omitempty"`
+	// Level - READ-ONLY; The level code, which only appears in the response.
+	Level common.InstanceViewStatus_StatusLevelType `json:"level,omitempty"`
```

Comment:
We should not use the grpc enums. We would want to use string enums here. Can we define them here